plugins/modules/scc_product.py

```diff
@@ -72,6 +76,22 @@ def main():
     module.task_timeout = 4 * 60
 
     with module.api_connection():
+        scc_version = '1.0.0'  # fallback
+        try:
+            statuses = module.foremanapi.resource('ping').call('statuses')
```

Comment:
Didn't we call ping in the `api_connect` process already? Maybe we can "Just" cache the result there?

This would be fine in a subsequent PR, however.

Reply:
it's not `ping`, it's `statuses`, different (more expensive) API